cypress/integration/CreateModels/EndSession.spec.js

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+import * as models from '../../support/Models'
+import * as modelPage from '../../support/components/ModelPage'
+import * as actions from '../../support/Actions'
+import * as editDialogModal from '../../support/components/EditDialogModal'
+import * as train from '../../support/Train'
+import * as trainDialogsGrid from '../../support/components/TrainDialogsGrid'
+import * as helpers from '../../support/Helpers'
+
+const preliminaryTrainingDescription = 'Preliminary Training to cause some expected behaviors in future Train Dialogs'
+
+describe('End Session - Create Model', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Should create a new model', () => {
+    it('Create a model to test against', () => {
+      models.CreateNewModel('z-EndSession')
+    })
+
+    it('Should create three Actions', () => {
+      actions.CreateNewActionThenVerifyInGrid({ response: "Hello" })
+      actions.CreateNewActionThenVerifyInGrid({ response: "Okay" })
+      actions.CreateNewActionThenVerifyInGrid({ type: 'END_SESSION', response: "Goodbye" })
+    })
+  })
+
+  context('1st Train Dialog', () => {
+    it('Should create a new Train Dialog', () => {
+      modelPage.NavigateToTrainDialogs()
+      cy.WaitForTrainingStatusCompleted()
+      train.CreateNewTrainDialog()
+    })
+
+    it('Should train model to respond to "Hi"', () => {
```

Comment:
Only chose this as example but "'Should train model to respond to "Hi"'" don't seem like tests. Just steps to setup the larger test which is to verify behavior of EndSession actions.

You can't tell what this is testing by reading it.  As I've commented on previous PR I think the text should say what it's asserting.

Reply:
True, let me see what can do.

Reply:
So yes it is true that the wording is not so much test verification oriented, even though what it says is what is verified, it is more setup oriented because that is what it is doing. The wording as a whole, along with all the other "it()" calls makes sense, it accurately describes what is going on.

I really am not sure of how to make it better. I tried, put in some effort, but it only made it worse in my opinion. Also I really don't believe in that philosophy of yours, it is too restrictive. I don't believe it can be followed 100%. So I think this is something we should work out in our test meeting.